### ISSUE TITLE
Paper prototyping & offline skits

### ISSUE THREAD

**Issue by dunebuggie** [chadsansing/critical-web-literacy-curriculum#2]
# Critical Web Literacy Module | Content Template - DRAFT
## Overview

**Module Title:** Intro to Critical Web Literacy

**Activity Title:** Paper prototyping & offline skits

**Image:** [image web address or URL]

**Image credit:** [license + author name linked to the page where the image lives]

**Made by:** Shreyas & Mozilla

**Time:** 45 minutes

**Activity X of Y** [this can be set later]

**Participate on the Web** 

**21st Century Skills:** [include all applicable skills]:
- [ ] Collaboration
- [ ] Communication
- [ ] Creativity

**Web Literacy Skills:** 
- [ ] Design
- [ ] Compose
- [ ] Revise
- [ ] Remix
- [ ] Connect
- [ ] Open Practice
- [ ] Contribute
- [ ] Share

**Learning Objectives:** 
- Learn the functioning of computer related concepts in a fun way

**Audience:** [skill level + role, like “beginner web user;” try to name a specific audience or two rather than listing all possible audiences]
Beginner web user

**Materials:** [include both online and offline materials needed to complete your activity]
- Pen
- Paper
- Scissors
## Preparation

What should the facilitator do before teaching this lesson?

Understand and acknowledge the importance of story telling in different setups- be it getting new contributors to an open source project or get students to understand a concept, story telling is a really powerful tool, if used in the right way, can yield great results. In this session, the story telling would be through a skit. 
## Introduction

The group is first introduced to each other and their facilitator. We then do an ice breaker for 3 minutes with the following open questions to the group:
1. When was the first time you've used the internet?
2. How often do you use the internet?
3. What sites do you most frequent

This helps us to understand our audience better and give them examples that they can relate better to. 
## Offline Activity

Ask the question to the group-
1. How many of you download content using torrents? 
In the first half of the session, we're going to understand how torrents work, with the help of a small skit.

Assemble the learners into group of 5. Assign them the following roles:
- Computer User(CU)
- Torrent file with Tracker(TFWT)
- Peers (3) (P)

The computer user searches the internet for a torrent file. Be sure to communicate clearly about copyrighted content. For the skit, we would be using the example of a file free of copyright. The CU acts as if he is searching for the file. This is when the "Torrent file with tracker" comes into the scene. TFWT then finds the file name in the database and identifies the Peers who are currently having the file and seeding it. Now the facilitator explains the concept of Seeding and Leeching. 

Seeding is when you download the file and continue to upload it so that someone else downloading the file can benefit by receiving a part of the file from you.

Leeching is when you download the file and don't seed. 

The 3 peers give some part of the file. This could be denoted in the form of a paper cutout in shapes with the marking P1, P2, and P3. Upon downloading these parts separately, they are then assembled together at the CU. This can be enacted by joining the parts of the puzzle paper cut out p1, p2 and p3. Another add-on could be multiple copies of the puzzle and CU could also give out pieces to p1, p2 and p3 so that at the end of the skit, they too have a complete piece. 

Find out other similar skit topics that can be used to explain web related concepts or functioning of a website/app that the learners use the most. Examples:
- How e-commerce website works
- How Wikipedia is such a large information repository
- What are the components of a URL and how it works

Prototype these skit ideas on a paper with diagrams, roles and connections between these roles. 
## Reflection

What have we learned today?
How many of you would prefer this kind of learning over theory classes of these concepts? 
How can we improve on this and add on to involve more people?

---

**Comment by chadsansing** [comment]
@dunebuggie, I love the torrent performance skit. This seems like a great way to model torrenting, and I bet we could also build out skits and games about peer-to-peer connections in general. Thank you!

Is the idea to follow up the performances with the paper-prototypes? Are those the storytelling pieces? We might benefit from some more explanation and instruction for that part of the activity. For example, we could call out performing and paper-prototyping as two different ways to tell stories about the web, or we could use recordings of the skits or documentation of the paper-prototypes as files to upload or to share with others face-to-face as stories about the web.

Does that make sense?

**Comment by dunebuggie** [comment]
@chadsansing Yes! Absolutely. I love the idea of using recordings of skits/documentation of the paper prototypes as files that we can later upload. Face to face sharing is also something that can be done. Perhaps these could be done based on the questions: 
1. When was the first time you've used the internet?
2. How often do you use the internet?
3. What sites do you most frequent?

This is something that I love that we do at Mozilla events where the facilitator asks questions and then we all discuss about it with each other. The outcomes usually are really great considering the diversity in our backgrounds at these events.

**Comment by chadsansing** [comment]
That sounds awesome, @dunebuggie - maybe that's an opportunity to loop back around from the beginning so participants:

1) Answer those questions.
2) Practice planning and performing through the torrent skit.
3) Prototype and share their own stories from the beginning using what they've learned.

I don't know - lots of great ideas here. How might you order them best?

**Comment by dunebuggie** [comment]
Yep. 1->2->3 is definitely the way to go.  👍🏻
